docker-orchestration/osrm-api-oasisd/docker-entrypoint.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash -xe
+BUILD_PATH=${BUILD_PATH:="/osrm-build"}
```

Comment:
Maybe put the contents into `osrm-backend/docker-entrypoint.sh` directly? I think we don't need an separate docker only for some scripts..

Reply:
Agrees that this is just script, I think later this one could merge with `osrm-api`'s `docker-entrypoint.sh`

Reply:
`Readme` also can be merged into `osrm-backend` docker's `readme` as a sub section or sub doc since it's describing how to use the service.  Then we can have one image but multiple binaries/scripts. Different services startup from one image but different entrypoint part/binary.